.travis.yml

```diff
@@ -6,5 +6,5 @@ jdk:
 before_script:
   - mysql -e "create database IF NOT EXISTS cromwell_test;"
 
-script: sbt -Dbackend.shared-filesystem.localization.0=copy clean coverage nodocker:test
+script: sbt -Dbackend.shared-filesystem.localization.0=copy clean coverage nodocker:test assembly
```

Comment:
Yes, yes it will slow down tests. "A lot" is relative. I'll leave it to reviewers to decide if we want to leave it in or not.